ykman/cli/otp.py

```diff
@@ -524,5 +534,8 @@ def settings(ctx, slot, enter, pacing, force):
     except YkpersError as e:
         _failed_to_write_msg(ctx, e)
 
+    if new_access_code is not None:
```

Comment:
Any reason not to do `if new_access_code:` instead of `if new_access_code is not None:` ?

Reply:
Not really... It would be if `new_access_code` could be empty, but that's not a valid value, so `if new_access_code:` should be functionally equivalent in this case. I'll change it.

Reply:
Actually, I realized only one of the two occurrences can be changed. If the first is `if new_access_code:`, then the parsing step is never entered, the program accepts the input and does nothing.